src/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+list(APPEND fortran_src
+  module_fv3gfs_ncio.f90
+)
+
+if(APPLE)
+  add_compile_definitions(APPLE)
+elseif(UNIX)
+  add_compile_definitions(LINUX)
+endif()
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "-g -traceback -convert big_endian -assume byterecl ${CMAKE_Fortran_FLAGS}")
+  set(CMAKE_Fortran_FLAGS_RELEASE "-O2")
+  set(CMAKE_Fortran_FLAGS_DEBUG "-ftrapuv")
+elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^(GNU)$")
+  set(CMAKE_Fortran_FLAGS "-g -fbacktrace -fconvert=big-endian ${CMAKE_Fortran_FLAGS}")
+  set(CMAKE_Fortran_FLAGS_RELEASE "-O2")
+  set(CMAKE_Fortran_FLAGS_DEBUG "-ggdb -Wall")
+endif()
+
+set(lib_name ${PROJECT_NAME})
+set(module_dir "${CMAKE_CURRENT_BINARY_DIR}/include")
+
+add_library(${lib_name} STATIC ${fortran_src})
+add_library(${PROJECT_NAME}::${lib_name} ALIAS ${lib_name})
+
+set_target_properties(${lib_name} PROPERTIES Fortran_MODULE_DIRECTORY
+    ${module_dir})
+
+target_include_directories(${lib_name} INTERFACE $<BUILD_INTERFACE:${module_dir}>
+                                                 $<INSTALL_INTERFACE:include>)
+
+target_link_libraries(${lib_name} PRIVATE NetCDF::NetCDF_Fortran)
+
+list(APPEND LIB_TARGETS ${lib_name})
+
+install(DIRECTORY ${module_dir} DESTINATION ${CMAKE_INSTALL_PREFIX})
+
+install(
+  TARGETS ${LIB_TARGETS}
+  EXPORT ${PROJECT_NAME}Exports
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+### Package config
+include(CMakePackageConfigHelpers)
+set(CONFIG_INSTALL_DESTINATION lib/cmake/${PROJECT_NAME})
```

Comment:
This should also be `CMAKE_INSTALL_LIBDIR`. It's wrong in other libraries too, but may as well fix it here to start with.

Reply:
done